data/constants.ts

```diff
@@ -1,2 +1,2 @@
 export const REPO_URL = 'ajnart/homarr';
-export const CURRENT_VERSION = 'v0.2.0';
+export const CURRENT_VERSION = 'v0.3.0';
```

Comment:
what is this file for? Is it used in `yarn build` or while nextjs is running?

Reply:
![image](https://user-images.githubusercontent.com/71191962/168486667-13707979-a156-4a1c-8f66-2410548094fc.png)

Reply:
is it needed after `yarn build` is run?